formbuilder/containers/builder/FormEdit.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
+import FormContainer from "../../containers/builder/FormContainer";
+import {getFormID} from "../../utils";
+
+export default class FormEdit extends Component {
+
+  componentDidMount() {
+    // If the schema is empty, then try to load the schema from the
```

Comment:
This comment is not finished :-)

Reply:
Looks like I did a copy/paste from UserForm.js!